test/EFCore.MySql.IntegrationTests/Tests/Models/ExpressionTest.cs

```diff
@@ -294,6 +301,33 @@ public async Task MySqlStartsWithOptimizedTranslator()
             Assert.False(result.NotStartsWith);
         }
 
+        [Fact]
+        public async Task MySqlMatchTranslator()
+        {
+            await Task.FromResult(1);
+            var result = _db.Products.Where(p => p.Name.Match("Product*"));
+
+            Assert.True(result.Any());
+        }
+
+        [Fact]
+        public async Task MySqlMatchInBooleanModeTranslator()
+        {
+            await Task.FromResult(1);
+            var result = _db.Products.Where(p => p.Name.MatchInBooleanMode("Product*"));
+
+            Assert.True(result.Any());
+        }
+
+        [Fact]
+        public async Task MySqlMatchWithQueryExpansionTranslator()
+        {
+            await Task.FromResult(1);
+            var result = _db.Products.Where(p => p.Name.MatchWithQueryExpansion("Product*"));
+
+            Assert.True(result.Any());
+        }
+
```

Comment:
Modify test names in accordance to the other tests in the test suite. Also assert for the generated SQL. This should be moved to the functional test project.

Reply:
Removed integration tests, will add functional tests when i'll understand how to create custom context with my index and test it. Also not sure about phrase **Also assert for the generated SQL**. Tests i have done were basic Read/Write tests to/from db, gonna look into it.

Reply:
> **Also assert for the generated SQL**. Tests i have done were basic Read/Write tests to/from db, gonna look into it.

I just mean to check the generated SQL against the expected one. E.g. all tests from the `SimpleQueryMySqlTest` class do this by calling the `AssertSql()` method at the end.

Reply:
You said its better to go **EF.DbFunctions.Like** approach. So i found `DbFunctionsMySqlTest.MySql.cs` file with tests for **LIKE** operation, however, all of thouse tests use some kind of standard context and i plan on creating my own with **FullTextIndex** declared there.
What name would be appropriate for test class with **MATCH** operations, which are **DbFunctions** too, something like `DbFunctionsExtensionsMySqlTest.MySql` is ok ?

Reply:
The name `MatchQueryMySqlTest` that you used is fine.